addons/binding/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/json/SenecHomeEnergy.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.senechome.internal.json;
+
+import java.io.Serializable;
+
+/**
+ * @author Steven.Schwarznau
+ *
+ */
+public class SenecHomeEnergy implements Serializable {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = -6171687327416551070L;
+
+    private String GUI_HOUSE_POW;
```

Comment:
```suggestion
    @SerializedName("GUI_HOUSE_POW")
    public String guiHousePow;
```

We usually use camelCase for fieldnames which are not static. You could also remove the getters and setters and make the fields public. This reduces a lot of code. Remove most empty lines in files.

Also applies to the other model files.